lib/l10n/intl_vi.arb

```diff
@@ -1413,659 +1413,596 @@
-  "@sign_in_to_self_hosted_instance": {
+  "this_feature_not_supported": "Tính năng này hiện chưa được hỗ trợ",
+  "@this_feature_not_supported": {
     "type": "text",
-    "placeholders_order": [],
     "placeholders": {}
   },
-  "do_not_have_an_account": "Don't have an account?",
-  "@do_not_have_an_account": {
+  "edit": "Chỉnh sửa",
+  "@edit": {
     "type": "text",
-    "placeholders_order": [],
     "placeholders": {}
   },
-  "sign_up": "Sign up",
-  "@sign_up": {
+  "save": "Lưu",
+  "@save": {
     "type": "text",
-    "placeholders_order": [],
     "placeholders": {}
   },
-  "sign_in_with_your_credentials": "Sign in with your credentials",
-  "@sign_in_with_your_credentials": {
+  "yes": "Đồng ý",
+  "@yes": {
     "type": "text",
-    "placeholders_order": [],
     "placeholders": {}
   },
-  "sign_in_with_your_sso": "Sign in with your SSO",
-  "@sign_in_with_your_sso": {
+  "no": "Không",
+  "@no": {
     "type": "text",
-    "placeholders_order": [],
     "placeholders": {}
   },
-  "please_input_your_url": "Please input your URL",
-  "@please_input_your_url": {
+  "the_upload_request_has_been_updated_successfully": "Cập nhật Yêu cầu tải lên thành công!",
+  "@the_upload_request_has_been_updated_successfully": {
     "type": "text",
-    "placeholders_order": [],
     "placeholders": {}
   },
-  "this_feature_not_supported": "This feature is not currently supported",
-  "@this_feature_not_supported": {
+  "the_upload_request_has_been_updated_failed": "Cập nhật Yêu cầu tải lên thất bại",
+  "@the_upload_request_has_been_updated_failed": {
     "type": "text",
-    "placeholders_order": [],
     "placeholders": {}
   },
-  "drive": "Drive",
+  "drive": "Ổ đĩa",
   "@drive": {
     "type": "text",
-    "placeholders_order": [],
     "placeholders": {}
   },
-  "loading": "Loading",
+  "loading": "Đang xử lý",
   "@loading": {
     "type": "text",
-    "placeholders_order": [],
     "placeholders": {}
   },
-  "create_new_workgroup_or_drive": "Create new workgroup or drive",
-  "@create_new_workgroup_or_drive": {
+  "canceled": "Huỷ bỏ",
+  "@canceled": {
     "type": "text",
-    "placeholders_order": [],
     "placeholders": {}
   },
-  "back": "Back",
+  "back": "Quay lại",
   "@back": {
     "type": "text",
-    "placeholders_order": [],
     "placeholders": {}
   },
-  "login_message_center": "Join +100 million active users using LinShare to store and share files with ease",
+  "login_message_center": "Tham gia cộng đồng +10000 người dùng LinShare\n giúp lưu trữ và chia sẻ tài liệu dễ dàng hơn",
   "@login_message_center": {
     "type": "text",
-    "placeholders_order": [],
     "placeholders": {}
   },
-  "login_message_bottom": "By default you connect to Linagora servers. If you are not sure about this setting, please contact your technical specialist.",
+  "login_message_bottom": "Kết nối mặc định tới máy chủ Linagora.\nNếu bạn không chắc chắn, hãy liên hệ với\n hỗ trợ kỹ thuật của bạn.",
```

Comment:
Remove the carriage return `\n` to display it on small screens for a better look.